### DISCUSSION TITLE
NA allowance on JenniNiku/gllvm?

### DISCUSSION THREAD

**Discussion by gerverska** [JenniNiku/gllvm#183] [Q&A] [answered]
Hello there,

I was wondering if allowing NAs in the response would ever make it to the `JenniNiku` fork of `gllvm`. Back on issue #34, it seemed that `BertvanderVeen/gllvm` had functionality for dealing with NAs in the response.

Are there any issues with this feature that have led to its omission from the `JenniNiku/gllvm`, or has its inclusion just not been a priority? Just curious in case there is something wrong with the feature.

Thanks!

**Comment by BertvanderVeen** [comment, marked as answer]
No, the feature is present in the main branch, and has not been omitted. A recent change of mine in the package broke some of this (vectorization of residual calculations), but I have addressed it in a new commit. Let me know if it works for you.

**Reply by gerverska** [reply]
Oh, awesome--I'll give it a go and let you know!
